packages/strapi-plugin-documentation/controllers/Documentation.js

```diff
@@ -140,15 +135,12 @@ module.exports = {
     const { error } = ctx.query;
 
     try {
-      const layout = fs.readFileSync(
-        path.join(__dirname, '..', 'public', 'login.html')
-      );
-      const $ = cheerio.load(layout);
+      const layout = fs.readFileSync(path.join(__dirname, '..', 'public', 'login.html'));
+      const filledLayout = _.template(layout)({
+        actionUrl: `${strapi.config.server.url}${strapi.plugins.documentation.config['x-strapi-config'].path}/login`,
+      });
+      const $ = cheerio.load(filledLayout);
 
-      $('form').attr(
-        'action',
-        `${strapi.plugins.documentation.config['x-strapi-config'].path}/login`
-      );
       $('.error').text(_.isEmpty(error) ? '' : 'Wrong password...');
```

Comment:
Can we just set this in the template and avoid using cheerio in this function ?  Same above do we still need cheerio ?